### PR TITLE
resoling issues with userdetail table view

### PR DESCRIPTION
**Pull request by ogwuazors**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

**Review by Nahomtes — APPROVED** [review]
Bug is fixed, name is being displayed as "Last, First"

File: src/bento/userDetailViewData.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { EDIT } from './adminData';
+
+export const getColumnInfo = (removeRenderer, approvedRenderer, accessType) => [
```
Comment (iksheth) [inline src/bento/userDetailViewData.js:3]: @ogwuazors We have to keep custodian file as a simple JSON file in most of the cases. Is it possible to convert getColumnInfo function to Array? If not, we can discuss other options.

File: src/bento/userDetailViewData.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    label: 'Approved By',
+    options: {
+      customBodyRender: approvedRenderer,
+    },
+  },
+  {
+    name: 'remove',
+    label: 'Remove',
+    options: {
+      display: (accessType === EDIT),
```
Comment (iksheth) [inline src/bento/userDetailViewData.js:18]: @ogwuazors Same requirement like above. We need to make custodian configuration file simple. Can we move this comparison in view? If not, we can discuss other options as well.

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/bento/userDetailViewData.js (hunk 2 above)
Comment (ogwuazors) [reply src/bento/userDetailViewData.js:18]: working on this.

File: src/bento/userDetailViewData.js (hunk 1 above)
Comment (ogwuazors) [reply src/bento/userDetailViewData.js:3]: working on this

**Review by iksheth — APPROVED** [review]

**Review by iksheth — APPROVED** [review]

**Pull request merged**